ansible/roles/pg_standby/templates/restore_from_backup.sh.j2

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+BACKUP=$1
+
+read -p 'Are you sure you want to restore postgres to a past state [y/N]: ' -r confirm
+if [ $confirm != 'y' ]; then
+    exit 1
+fi
+read -p 'This will delete the current db. Are you SURE you are sure [y/N]: ' -r confirm2
```

Comment:
Yeah, it sounds scary on purpose but realistically the most probable scenario for running this command is on a new machine if the main db (and standby for prod) is hosed. I dont really see this being run on a machine with existing data, because those machines have backups already on the machine.